src/panels/calendar/ha-full-calendar.ts

```diff
@@ -278,6 +278,17 @@ export class HAFullCalendar extends LitElement {
       this.shadowRoot!.getElementById("calendar")!,
       config
     );
+
+    // set the calendar to today every day at midnight
+    (function daily(calendar) {
+      const m = new Date().setHours(24, 0, 0, 0);
```

Comment:
You should take timezones into consideration, the timezone used in Home Assistant might be different than the timezone of the browser.

In general, set your timezone to something different than your browser is using:
![CleanShot 2023-08-18 at 10 21 24](https://github.com/home-assistant/frontend/assets/5662298/16de78ee-81b0-44ba-b8b5-518e51669118)

In profile, select use server timezone:
![CleanShot 2023-08-18 at 10 22 42](https://github.com/home-assistant/frontend/assets/5662298/b2e5d418-5aef-421a-ab5c-5c6fde041114)

Reply:
Isn't the browser's "today" the one that matters though?

Reply:
Where is `today()` defined? I can't find it.

```
  private _handleToday(): void {
    this.calendar!.today();
    this._fireViewChanged();
  }
```

Reply:
It is a function of Full Calendar